combustion-android-ble/src/main/java/inc/combustion/framework/ble/uart/meatnet/NodeRequest.kt

```diff
@@ -68,9 +69,9 @@ internal open class NodeRequest(
             }
 
             // Message type
-            val typeByte = data[4]
+            val rawMessageType = data[4]
 
-            val messageType = NodeMessageType.fromUByte(typeByte) ?: return null
+            val messageType = NodeMessageType.fromUByte(rawMessageType)// ?: null
```

Comment:
looks like commented code that was intended to be removed

Reply:
Whoops, yes. Thanks for catching this.